content/filterx/filterx-otel/_index.md

```diff
@@ -0,0 +1,310 @@
+            log.observed_time_unix_nano = ${UNIXTIME};
+            log.body = ${MESSAGE};
+            log.severity_number = ${LEVEL_NUM};
+
+            # Update output
+            ${.otel_raw.log} = log;
+            ${.otel_raw.resource} = resource;
+            ${.otel_raw.scope} = scope;
+            ${.otel_raw.type} = "log";
+        };
+        destination {
+            # your opentelemetry destination settings
+        };
+    };
+    ```
+    <!-- FIXME do we need the Update output part in this case? -->
+
+## otel_logrecord reference {#otel-logrecord-reference}
+
+OpenTelemetry log records can have the following fields. (Based on the [official OpenTelemetry proto file](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/logs/v1/logs.proto).)
+
+### attributes
+
+|           |                                                  |
+| --------- | ------------------------------------------------ |
+| Type: | `otel_kvlist` |
+
+Attributes that describe the event. Attribute keys MUST be unique.
+
+### body
+
+The body of the log record. It can be a simple string, or any complex nested objects, such as lists and arrays.
+
+### flags
+
+|           |                                                  |
+| --------- | ------------------------------------------------ |
+| Type: | `int` |
+
+Flags as a bit field.
+
+### observed_time_unix_nano
+
+|           |                                                  |
+| --------- | ------------------------------------------------ |
+| Type: | `datetime` |
+
+The time when the event was observed by the collection system, in UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January 1970.
+
+### severity_number
+
+|           |                                                  |
+| --------- | ------------------------------------------------ |
+| Type: | `int` |
+
+The severity of the message as a numerical value of the [severity](#severity_text).
+<!-- FIXME include or link to the string mapping -->
+
+### severity_text
+
+|           |                                                  |
+| --------- | ------------------------------------------------ |
+| Type: | `string` |
+
+The severity of the message as a string:
+
+```
+SEVERITY_NUMBER_UNSPECIFIED = 0;
+SEVERITY_NUMBER_TRACE  = 1;
+SEVERITY_NUMBER_TRACE2 = 2;
+SEVERITY_NUMBER_TRACE3 = 3;
+SEVERITY_NUMBER_TRACE4 = 4;
+SEVERITY_NUMBER_DEBUG  = 5;
+SEVERITY_NUMBER_DEBUG2 = 6;
+SEVERITY_NUMBER_DEBUG3 = 7;
+SEVERITY_NUMBER_DEBUG4 = 8;
+SEVERITY_NUMBER_INFO   = 9;
+SEVERITY_NUMBER_INFO2  = 10;
+SEVERITY_NUMBER_INFO3  = 11;
+SEVERITY_NUMBER_INFO4  = 12;
+SEVERITY_NUMBER_WARN   = 13;
+SEVERITY_NUMBER_WARN2  = 14;
+SEVERITY_NUMBER_WARN3  = 15;
+SEVERITY_NUMBER_WARN4  = 16;
+SEVERITY_NUMBER_ERROR  = 17;
+SEVERITY_NUMBER_ERROR2 = 18;
+SEVERITY_NUMBER_ERROR3 = 19;
+SEVERITY_NUMBER_ERROR4 = 20;
+SEVERITY_NUMBER_FATAL  = 21;
+SEVERITY_NUMBER_FATAL2 = 22;
+SEVERITY_NUMBER_FATAL3 = 23;
+SEVERITY_NUMBER_FATAL4 = 24;
+```
```

Comment:
These correspond to the `severity_number` field.

Reply:
Also, it would be sensible to mention `otel.<severity>` enums here, as it makes it easier to configure this field for the user. Setting literal integers is not the cleanest or easiest.